sycl/include/CL/sycl/detail/rocm_definitions.hpp

```diff
@@ -0,0 +1,24 @@
+//==------------ cuda_definitions.hpp - SYCL CUDA backend ------------------==//
```

Comment:
```suggestion
//==------------ rocm_definitions.hpp - SYCL ROCM backend ------------------==//
```
ROCM or HIP?

Reply:
If CUDA here refers to the runtime, then it should be ROCm. If CUDA here refers to the front-end programming language, then it should be HIP. And I am not very clear about this.